src/os/filestore/FileStore.cc

```diff
@@ -3715,9 +3697,9 @@ int FileStore::_clone(const coll_t& cid, const ghobject_t& oldoid, const ghobjec
   _set_replay_guard(**n, spos, &newoid);
 
  out3:
-  lfn_close(n);
+  dout(20) << __func__ << "out3" << dendl;
```

Comment:
please remove these logging messages and take this chance to consolidate these labels into a single one.